### PR TITLE
[VALIDATE CANCEL] Reduce the pop in size and unable users to see details

### PR DESCRIPTION
**Pull request by cmarianhodis**
To be remembered: the new labels are translated only for english and french, for rest of the languages they will need to be translated.

### CODE REVIEW

File: src/tb/component/revisionselector/main.js [hunk 1]
```diff
@@ -87,6 +89,18 @@ define(
                 },
 
                 /**
+                 * Bind events
+                 */
+                bindEvents: function () {
```
Comment (fkroockmann) [inline src/tb/component/revisionselector/main.js]: On this case you can remove var statement for optimize memory cause you use var one time, you can do it like that: 

````
jQuery(this.seeDetailsClass).on('click', function () {
    jQuery(this.revisionListClass).toggle();
});
````

**Comment by cmarianhodis** [comment]
Modifications made, also, like for the Page issue, I kept only the keys for the languages that I didn't had the translations

**Comment by fkroockmann** [comment]
dont keep the keys else user dont see anything

**Comment by cmarianhodis** [comment]
Removed here also

**Pull request merged**